docs/gettingstarted/index.rst

```diff
@@ -257,7 +257,7 @@ Getting started with Flocker
 
       .. code-block:: console
 
-         you@laptop:~$ flocker-deploy deployment-node1.yml fig.yml
+         you@laptop:~$ flocker-deploy 172.16.255.250 deployment-node1.yml fig.yml
```

Comment:
:question: Not a new issue - fix or file an issue - the above text references `deployment.yml` but that is not the name of the file.